server/build.gradle.kts

```diff
@@ -79,11 +79,11 @@ val copyDockerFolder: TaskProvider<Copy> =
         into(dockerBuildRootDirectory)
     }
 
-// @todo(#343) - createProductionDotEnv is temporary and used by the suites,
+// @todo(#343) - createDotEnv is temporary and used by the suites,
 //  once the suites no longer rely on the .env file from the build root we
 //  should remove this task
-val createProductionDotEnv: TaskProvider<Exec> =
-    tasks.register<Exec>("createProductionDotEnv") {
+val createDotEnv: TaskProvider<Exec> =
```

Comment:
This is planned to be removed. I do not think we need to change anything here, The update shell could be called by the command line from gradle, IMO this change is unnecessary and would entail additional maintenance.